.devops/pagopa-deploy-pipelines.yml

```diff
@@ -159,7 +159,7 @@ stages:
               yarn global add newman
             displayName: 'newman installation'
           - script: |
-              newman run ../api-tests/buyerbanks.api.tests.json --environment=../api-tests/dev.envs.json --reporters cli,junit --reporter-junit-export Results/buyerbanks-api-TEST.xml
+              newman run https://raw.githubusercontent.com/pagopa/pagopa-functions-buyerbank/master/api-tests/buyerbanks.api.tests.json --environment=https://raw.githubusercontent.com/pagopa/pagopa-functions-buyerbank/master/api-tests/dev.envs.json --reporters cli,junit --reporter-junit-export Results/buyerbanks-api-TEST.xml
```

Comment:
IIRC paths are relative to the repo root, so if you want you should be able to refer to the locally checked out JSON with `./api-tests/buyerbanks.api.tests.json` (same for the other)
Am I missing something?

Reply:
I've used the same approach used [here](https://github.com/pagopa/pagopa-functions-checkout/blob/24d8b5a037c9f2e9d350626e34816faf56515b07/.devops/pagopa-deploy-pipelines.yml#L159), I don't know if it can be done using relative import during pipeline execution since not all files are part of the artifact